backend/dataall/cdkproxy/stacks/sagemakerstudio.py

```diff
@@ -1,126 +1,19 @@
 import logging
 import os
-import pathlib
-from constructs import Construct
 from aws_cdk import (
     cloudformation_include as cfn_inc,
-    aws_ec2 as ec2,
-    aws_iam as iam,
-    aws_kms as kms,
-    aws_lambda as _lambda,
-    aws_sagemaker as sagemaker,
-    aws_ssm as ssm,
-    CustomResource,
-    Duration,
-    NestedStack,
     Stack
 )
-from botocore.exceptions import ClientError
 from .manager import stack
 from ... import db
 from ...db import models
 from ...db.api import Environment
-from ...aws.handlers.parameter_store import ParameterStoreManager
-from ...aws.handlers.sts import SessionHelper
-from ...aws.handlers.sagemaker_studio import (
-    SagemakerStudio,
-)
 from ...utils.cdk_nag_utils import CDKNagUtil
 from ...utils.runtime_stacks_tagging import TagsUtil
 
 logger = logging.getLogger(__name__)
 
 
-class SageMakerDomain(NestedStack):
```

Comment:
Why should we delete `SageMakerDomain` ? If we decided not to go with the nested stack and rollback it, I'd still leave this class, remove only `NestedStack, changed init to `create_ml_studio_part` (or something like that) and keep using it :)